lua/obsidian/commands/new_from_template.lua

```diff
@@ -0,0 +1,45 @@
+local util = require "obsidian.util"
+local log = require "obsidian.log"
+
+---@param client obsidian.Client
+return function(client, data)
+  ---@type obsidian.Note
+  if not client:templates_dir() then
+    log.err "Templates folder is not defined or does not exist"
+    return
+  end
+
+  local note
+  if data.args:len() > 0 then
```

Comment:
This would make this a little more robust, in case `data.args` is `nil`.
```suggestion
  if data.args and data.args:len() > 0 then
```

Reply:
Done